util/templates/template_modbus.go

```diff
@@ -75,8 +75,8 @@ func (t *Template) ModbusValues(renderMode int, values map[string]interface{}) {
 
 			switch p.Name {
 			case ModbusParamNameId:
-				if modbusParam.ID != 0 {
-					defaultValue = strconv.Itoa(modbusParam.ID)
+				if modbusParam.ID != nil {
```

Comment:
@JeppeKnockaert thank you for this change. Since the modbus interface is a bit finickey, I don't want to apply this to reduce risk of breaking things. In that case I'd rather remove the "1" default from the defaults.yaml.

/cc @premultiply

Reply:
I understand, I think the added documentation will already be enough to help people out.